samples/go/photo-dedup/job/dedupjob.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package job
```

Comment:
Whatever code is being shared between these two jobs seems like it would be just as useful to other jobs, and should be outside this directory.